src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketBuildStatusNotifications.java

```diff
@@ -150,34 +159,48 @@ private static BitbucketSCMSource lookUpBitbucketSCMSource(final SCMSourceOwner
         }
         return null;
     }
+    
+    private static void sendNotifications(Run<?, ?> build, TaskListener listener)
+    {
```

Comment:
Please, follow the code style in already existent code base (no newline for braces).